packages/execution-environments/src/common/lockdown/lockdown.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable-next-line import/no-unassigned-import */
+import 'ses';
```

Comment:
We should get rid of this `ses` import, too. For future reference, SES only needs to be imported once, and it mutates the global environment (realm) automagically when it's imported.